jormungandr/src/network/p2p/layers/preferred_list.rs

```diff
@@ -0,0 +1,91 @@
+pub use poldercast::Address;
+use poldercast::{GossipsBuilder, Layer, NodeProfile, Nodes, ViewBuilder};
+use rand::{seq::SliceRandom as _, Rng as _, SeedableRng};
+use rand_chacha::ChaChaRng;
+use serde::{Deserialize, Serialize};
+
+const DEFAULT_PREFERRED_VIEW_MAX: usize = 20;
+
+#[derive(Debug, Serialize, Deserialize, Default)]
+pub struct PreferredListConfig {
+    #[serde(default)]
+    view_max: PreferredViewMax,
+
+    #[serde(default)]
+    peers: Vec<Address>,
```

Comment:
I would use a `HashSet` here unless we want the order to matter. 
Duplicates make no sense, or do them?

Reply:
This is a good idea, we can't do that yet now, as we cannot use just the `Address` here, we need the `Id` too (we are pre-0.12 era of poldercast still). I have updated the comment to remind to use a `HashSet` when we update poldercast